### ISSUE TITLE
"region" Importance in creating a s3object

### ISSUE THREAD

**Issue by jammy-srikanth** [aws/aws-sdk-js#768]
previously we were using the below code to create a s3 object to a s3 bucket region(us-west-1).
```
var s3 = new aws.S3({
            "region": process.env.S3_REGION
        });
```
Now,we want to upload to multiple regions  with single object so i tried commenting "region" in the constructor and uploaded to different regions like eu-west-1,us-west-2 etc everything works fine.

i have some queries to be clarified before i move this code to production
1)What is the importance of region ? will it cause any issues if i remove it?
2)would the upload speed is affected?

**Comment by chrisradek** [comment]
@jammy-srikanth 
You'll probably get a more comprehensive answer by asking on the S3 AWS forums.  As far as potential issues, you may run into an issue when using sigv4 signing.  For example, if your default region was `us-east-1`, and you were trying to create a new object in `eu-central-1` - a region that only supports sigv4 - then there would be an authorization failure because the region is used when creating a v4 signature.  A recent update to the SDK was made available with the latest version (2.2.13) so that when this type of failure is encountered, the request will automatically be retried with the correct region.

Let me know if this helps!

**Comment by chrisradek** [comment]
I'm going to close this issue as this question is better directed at the S3 service team on the forums. Feel free to comment or re-open if you'd like to discuss further.
